ops/backup.go

```diff
@@ -0,0 +1,89 @@
+package ops
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/tecbot/gorocksdb"
+)
+
+// Current is the file which terminates recursive loops
```

Comment:
@aayushKumarJarvis I guess the larger reason why we're using `CURRENT` is to identify a rocksdb store.